notebooks/scripts/example_study.py

```
@@ -17,6 +17,29 @@
+        (i for i, part in enumerate(path_parts) if not part.startswith("tmp")), 1
+    )
+    return Path(*path_parts[start_index:])
+
+
+# %%
+remove_tmp_prefixes("tmp/tmp223/sas")
+
+# %%
+with open("../ehr2vec/azure_run/credentials.json", "r") as f:
+    credentials = json.load(f)
+credentials
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)